src/App.tsx

```diff
@@ -76,8 +77,9 @@ function App() {
             onRangeChange={handleRangeChange}
             isLoading={false}
             data={filteredData}
-            onItemClick={handleItemClick}
+            onTileClick={handleItemClick}
```

Comment:
Please change the method name to `handleTileClick`, for consistency

Reply:
Changed